packages/utils/src/index.ts

```diff
@@ -16,7 +16,7 @@ import GlobalContext, { GlobalProvider } from "./contexts/Global";
 import type { MarqueeWindow, ContextProperties } from './types';
 
 const defaultChannel = 'vscode.marquee';
-const theme = createTheme(calculateTheme());
+const theme = createTheme(adaptV4Theme(calculateTheme()));
```

Comment:
In `calculateTheme.ts` we already return:
```ts
return createTheme(adaptV4Theme(newTheme));
```

It seems we call it twice now, right?

Reply:
This is because before we called `createTheme` in both places, see [here](https://github.com/stateful/vscode-marquee/blob/main/packages%2Futils%2Fsrc%2Findex.ts#L19) and [here](https://github.com/stateful/vscode-marquee/blob/main/packages%2Futils%2Fsrc%2FcalculateTheme.ts#L79). I was just being defensive, since I wasn't sure what exactly what the reasoning was. 

Maybe it's safe to only call `createTheme` and `adaptV4Theme ` in `calculateTheme`?

Reply:
@christian-bromann I just removed it now - if there was a reason for calling `createTheme` twice, let me know and I reverse it.